src/App.tsx

```
@@ -72,67 +55,104 @@ function nextStep(current: Pos, target: Pos, visited: Pos[]) {
+            ) {
+              enemy.status = Status.Disabled;
+              enemy.focus = false;
+              console.log(life, newLife)
+              newLife -= 1 
+              console.log(life, newLife)
+            } else if (enemy.status == Status.Active) {
+                enemy.timeActivated += delta
+                if (enemy.timeActivated >= MOVEMENT_THRESHOLD){
+                      const allPositions = enemies.reduce((acc: Pos[], {position, status}: {position: Pos, status: Status})=> {
+                          if ([Status.Active, Status.Inactive].includes(status))
+                              return [...acc, position]
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)